pythonforandroid/bootstraps/webview/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -78,6 +78,8 @@
             {%- if args.intent_filters -%}
             {{- args.intent_filters -}}
             {%- endif -%}
+            {% for m in args.activity_meta_data %}
+            <meta-data android:name="{{ m.split('=', 1)[0] }}" android:value="{{ m.split('=', 1)[-1] }}"/>{% endfor %}
```

Comment:
It's nice to do this all in Jinja, but I would probably prefer to do this in `build.py` and substitute fully prepared statements.

Reply:
Good points! I was just copying the for loop in this same template for the application meta-data. Do you think I should update those along the way? Or prefer inconsistency?

Reply:
Oh, I skimmed the existing code too fast. I personally would like to clean up the old one, but upstream may feel different. I think I'd probably just leave the existing code alone.